modules/installation-osp-creating-control-plane-ignition.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-creating-control-plane-ignition_{context}"]
+= Creating control plane ignition files
+
+Installing {product-title} on {rh-openstack-first} on your own infrastructure requires control plane Ignition files. You must create multiple ignition files. 
+
+[NOTE]
+As with the bootstrap Ignition configuration, you must ensure that the hostname per control plane machine is set to an expected value.
+
+.Prerequisites
+
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
+** If the variable is not set, see *Creating the Kubernetes manifest and Ignition config files*.
+
+.Procedure
+
+. At a command line prompt, run the following Python script:
++
+----
+$ for index in $(seq 0 2); do
+    MASTER_HOSTNAME="$INFRA_ID-master-$index\n"
+    python -c "import base64, json, sys;
+ignition = json.load(sys.stdin);
+files = ignition['storage'].get('files', []);
+files.append({'path': '/etc/hostname', 'mode': 420, 'contents': {'source': 'data:text/plain;charset=utf-8;base64,' + base64.standard_b64encode(b'$MASTER_HOSTNAME').decode().strip(), 'verification': {}}, 'filesystem': 'root'});
+ignition['storage']['files'] = files;
+json.dump(ignition, sys.stdout)" <master.ign >"$INFRA_ID-master-$index-ignition.json"
+done
+----
+
+You now have three control plane Ignition files: <INFRA_ID>-master-0-ignition.json, <INFRA_ID>-master-1-ignition.json,
+and <INFRA_ID>-master-2-ignition.json.
```

Comment:
all of this files also need to be copied to the same directory where the playbooks (common.yaml) exists.  The playbooks look for the files in the same directory where they exists.

Reply:
If the json files are not in the correct directory the user will see an error like this one: 


fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, original message: could not locate file in lookup: newcluster-zvmqn-bootstrap-ignition.json"}